weblab_cg/data/tests/static_model_graphstate.pyx

```diff
@@ -181,8 +181,8 @@ cdef class TestModel(CvodeSolver):
 
         # Oxmeta names of output variables
         self.outputNames = []
-        self.outputNames.append('time')
-        self.outputNames.append('membrane_voltage')
+        self.outputNames.append('https://chaste.comlab.ox.ac.uk/cellml/ns/oxford-metadata#time')
```

Comment:
WL1 uses just the local names here, without the base URI part. What we should do eventually is update the `ModelWrapperEnvironment` so we can use a separate instance for each namespace defined by the protocol, and then we can use longer names here and let each environment wrap its respective subset. But until that happens, users just have to make sure not to use the same local name in different namespaces.

Reply:
Note that these should be local names *not* cmeta ids, even though they're sometimes the same!

Reply:
Thanks, have put these comments in the source code for now